independent-publisher-connectors/WeatherForecast/apiDefinition.swagger.json

```diff
@@ -0,0 +1,229 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "weatherforecast",
```

Comment:
@Haimantika The connector title should be free from grammatical and spell errors.

Docs - [Give your connector a title](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)